src/components/flex/_flex_item.scss

```diff
@@ -24,18 +24,47 @@
 
   /*
    * 1. We need the extra specificity here to override the FlexGroup > FlexItem styles.
-   * 2. FlexItem can be manually set to not grow if needed.
+   * 2. FlexItem can be manually set to not grow or shrink if needed (default allows grow and shrink).
+   * 3. FlexItem basis can be set to 'auto' (defaul use flex-basis: 0%)
+   * 4. FlexItem also can be given values for grow, shrink and basis:
+   *    - grow: 1..10
+   *    - shrink: 1..10
+   *    - basis: auto, max-content, min-content, fit-content
    */
+
   &.ouiFlexItem--flexGrowZero { /* 1 */
     flex-grow: 0; /* 2 */
     flex-basis: auto; /* 2 */
   }
 
-  @for $i from 1 through 10 {
-    &.ouiFlexItem--flexGrow#{$i} {
-      flex-grow: $i;
+  &.ouiFlexItem--flexShrinkZero { /* 1 */
```

Comment:
What do these numbered comments mean?

Reply:
It is used to refer to the comments above

 /*
   * 1. We need the extra specificity here to override the FlexGroup > FlexItem styles.
   * ...
   */